.travis.yml

```diff
@@ -1,4 +1,4 @@
 language: node_js
 node_js:
-  - 0.10
```

Comment:
You can just use `stable` to always track the latest version (which is actually `5.4.1` currently).